lib/ansible/module_utils/redfish_utils.py

```diff
@@ -317,15 +314,24 @@ def get_storage_controller_inventory(self):
         result["entries"] = controller_results
         return result
 
-    def get_disk_inventory(self):
+    def get_multi_storage_controller_inventory(self):
+        ret = True
+        entries = {}
+        for systems_uri in self.systems_uris:
+            inventory = self.get_storage_controller_inventory(systems_uri)
+            ret = inventory.pop('ret') and ret
+            entries[system_id] = inventory
```

Comment:
Will the `entries[system_id]` value be overridden on every iteration leaving just one entry in the `entries` dict?